src/actions/actualUser.actions.js

```diff
@@ -0,0 +1,41 @@
+import services from '../services/ServiceProvider';
+
+export const LOAD_ACTUAL_USER = 'LOAD_ACTUAL_USER';
+export const LOAD_ACTUAL_USER_REPOSITORIES = 'LOAD_ACTUAL_USER_REPOSITORIES';
+
+function parseData(repositories) {
+  return repositories.map(repository => {
+    return {
+      title: repository.name,
+      content: repository.description,
+    };
+  });
+}
+
+export function resetRepositories() {
+  return loadRepositories([]);
+}
+
+export function loadActualUser(user) {
+  return {
+    type: LOAD_ACTUAL_USER,
+    user,
+  };
+}
+
+export function loadRepositories(repositories) {
+  return {
+    type: LOAD_ACTUAL_USER_REPOSITORIES,
+    repositories,
+  };
+}
+
+export function  loadUserRepositories(userName) {
```

Comment:
Maybe it's silly, but there are double space between `function` and `loadUserRepositories`
```suggestion
export function loadUserRepositories(userName) {
```

Reply:
eslint should throw here ...

Reply:
we will add this rule